subsys/nrf_security/src/drivers/cracen/silexpk/src/ed25519.c

```diff
@@ -138,7 +139,7 @@ static inline int ed25519_decode_pt_x(const struct sx_ed25519_pt *pt)
 static inline void ed25519_pt_write_y(const struct sx_ed25519_pt *pt, char *ay)
 {
 	sx_wrpkmem(ay, pt->encoded, SX_ED25519_PT_SZ);
-	sx_wrpkmem_byte(&ay[SX_ED25519_PT_SZ - 1], pt->encoded[SX_ED25519_PT_SZ - 1] & 0x7f);
```

Comment:
I don't think it's safe to not use sx_wrpkmem_byte here

Reply:
Reverted to use sx_wrpkmem_byte